cartosky/core.py

```diff
@@ -95,7 +95,7 @@ def hpx2xy(self, hpxmap, pixel=None, nside=None, xsize=800,
 
     def smooth(self,hpxmap,badval=hp.UNSEEN,sigma=None):
         """ Smooth a healpix map """
-        healpix.check_hpxmap(hpxmap,None,None)
+        _ = healpix.check_hpxmap(hpxmap,None,None)
```

Comment:
This is probably something I don't know, but why do we want to capture the output?

Reply:
For that one is not necessary but for `draw_hpxmap` I thought that was the cleanest way to check whether something is a healsparse map or not. I guess I can just take out the `_ =` there